### PR TITLE
pdf_storage/pdf_support removal

### PR DESCRIPTION
**Pull request by RFrailich**

### CODE REVIEW

**Comment by david-yz-liu** [comment]
Hi @RFrailich, there are some changes in this commit which shouldn't be there (.ruby-version, Gemfile, dbout). You should also delete the keys from `markus_configurator.rb`

**Comment by RFrailich** [comment]
.ruby-version and dbout can go in git ignore?

**Comment by david-yz-liu** [comment]
No, you should not have staged them to be added to a commit.

However, at this point it would be easiest if you created another commit to revert those changes.

File: app/controllers/results_controller.rb [hunk 1]
```diff
@@ -333,8 +333,7 @@ def codeviewer
     # if dealing with a pdf file, get the number of images to display
     if @file.is_pdf?
```
Comment (david-yz-liu) [inline app/controllers/results_controller.rb]: @RFrailich It seems this whole block can be removed, as it is calculating something which doesn't make any sense. Please also look for uses of the variable `@nb_pdf_files_to_download` and see if you can delete any of those.

File: app/models/submission_collector.rb [hunk 2]
```diff
@@ -238,20 +238,9 @@ def manually_collect_submission(grouping, rev_num,
     self.safely_stop_child_exited = true
     self.save
 
-    #Let the child process handle conversion, as things go wrong when both
-    #parent and child do this.
-    start_collection_process do
```
Comment (david-yz-liu) [inline app/models/submission_collector.rb]: The call to `start_collection_process` should be kept, as it is what... starts the collection process. I would keep the do block, but remove the checks inside which refer to the unused variables.

Comment (RFrailich) [inline app/models/submission_collector.rb]: IS this still necessary? (from line 249)
    #setting is_collected here will prevent an sqlite lockout error when pdfs
    #aren't supported
      grouping.is_collected = true
      grouping.save

File: elec [hunk 3]
```diff
@@ -0,0 +1,46 @@
+                  List of relations
```
Comment (david-yz-liu) [inline elec]: @RFrailich can you remove this file as well?

**Comment by david-yz-liu** [comment]
Hi @RFrailich, it seems there are references to removed code in the tests, some of which are now failing. Can you please fix the tests as well?

**Pull request merged**